api/types/database.go

```diff
@@ -290,6 +292,14 @@ func (d *DatabaseV3) GetAdminUser() string {
 	return d.Metadata.Labels[DatabaseAdminLabel]
 }
 
+// GetOracle returns database privileged user information.
+func (d *DatabaseV3) GetOracle() *OracleOptions {
+	if d == nil {
+		return nil
+	}
```

Comment:
in what cases `d` is `nil`?

Reply:
I have borrowed the patter from gRPC [Get](https://github.com/gravitational/teleport/blob/7e6b941dcd73ae25abe618f313b28c02e542788e/gen/proto/go/teleport/lib/teleterm/v1/cluster.pb.go#L232)

But agree that this is corner case and the `d`  probably is never null.